File: frontend/app/Sala.js
export class Sala {
    constructor(id, naziv, ukupanBrojRedova, ukupanBrojSedista) {
        this.id = id;
        this.naziv = naziv;
        this.ukupanBrojRedova = ukupanBrojRedova;
        this.ukupanBrojSedista = ukupanBrojSedista;
    }

    get sedistaPoRed() {
        return Math.floor(this.ukupanBrojSedista / this.ukupanBrojRedova);
    }
}

File: APRIL - 2024/backend/Models/Sala.cs
using System.Text.Json.Serialization;

namespace backend.Models;

public class Sala{
    [Key]
    public int ID {get; set;}
    public required string Naziv {get; set;}
    public int UkupanBrojRedova {get; set;}
    public int UkupanBrojSedista {get; set;} // mora biti deljivo
    public List<Karta>? Karte {get; set;}
}